packages/templates/src/getTemplates.js

```diff
@@ -127,7 +127,7 @@ async function getTemplates(imageBaseUrl) {
 
   trackTiming();
 
-  return templates;
+  return templates.sort((a, b) => b.creationDate - a.creationDate);
```

Comment:
When using `toDate()` as per my other comment, we can make our date package re-export `compareAsc`/`compareDesc` from `date-fns` and use that for comparison here instead.